stress_test/sample_test_confs/docker/beryllium/beryllium_sb_active_scalability_multinet.json

```diff
@@ -68,6 +68,7 @@
 "interpacket_delay_ms":5000,
 
 "oftraf_rest_server_port":5555,
+"oftraf_venv_path":"/opt/venv_oftraf",

```

Comment:
why does it have to be exposed to the user? can't we make it transparent by setting it internally to some hardcoded path?